libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -39,6 +39,14 @@ bytes toBytes(uint8_t _b)
 	return bytes(1, _b);
 }
 
+enum class Flags: uint8_t
+{
+	Immutable = 0,
+	Mutable = 1,
+	MaximumFieldOmitted = 0,
+	MaximumFieldPresent = 1,
+};
+
```

Comment:
This can be done in several ways but each one has something I don't like about it:
- `enum class`: needs `static_cast<uint8_t>` to convert to a number.
- `enum`: constants would not be namespaced with `Flag::`
- loose `uint8_t constexpr`s: the compiler issues warnings about the ones that are not used in the code.
- `struct` with `uint8_t constexpr`s inside: a class without data inside feels like it should actually be a ``namespace``. And the compiler would probably warn about unused variables too.
- `namespace` with `uint8_t constexpr`s inside: again, warning about unused variables.